rorocloud/client.py

```diff
@@ -63,45 +63,78 @@ def delete(self, path):
 
     def jobs(self, all=False):
         if all:
-            return [Job(job) for job in self.get("/jobs?all=true")]
+            response = self.get("/jobs?all=true")
+            if response.status_code != 200:
+                self.handle_error(response)
```

Comment:
Please use `return self.handle_error(response)` to make it clear that the code below will never be executed.